docs/pages/setup/operations/ca-rotation.mdx

```diff
@@ -271,8 +282,17 @@ Host CA  initialized (mode: manual, started: Sep 20 01:44:36 UTC, ending: Sep 21
   type="warning"
   title="CA Pinning Warning"
 >
-  If you are using [CA Pinning](../admin/adding-nodes.mdx#untrusted-auth-servers) when adding new nodes, the CA pin will change after the rotation. Make sure you use the
-  *new* CA pin when adding nodes after rotation.
+  If you are using
+  [CA Pinning](../admin/adding-nodes.mdx#untrusted-auth-servers) when adding new
+  nodes, the CA pin will change after the rotation. Make sure you use the *new*
+  CA pin when adding nodes after rotation.
+</Admonition>
+
+<Admonition type="warning" title="Desktop Access">
+Teleport signs Windows Desktop certificates with the user certificate authority.
+If the user CA is rotated, the new certificate will need to be
```

Comment:
Rather than add the link on "exported", which could break the user's focus, I should we should add a "Read more about exporting the Teleport CA" link at the bottom of this admonition box.

Reply:
I am definitely not the expert, so I defer to you on the final decision, but in this case I think the link makes sense.

As a reader, I'm thinking "you're telling me I have to do this export thing, but I have no idea what that means or how to do it. Why make me search for it?"